### PR TITLE
Add initial geography support

### PR DESCRIPTION
**Pull request by wchargin**
This follows the form of a recent commit to `pgx-geos` for the same functionality.

### CODE REVIEW

File: pgxgeom_test.go [hunk 1]
```diff
@@ -115,6 +115,23 @@ func TestCodecScanValue(t *testing.T) {
...
+		tb.Helper()
+		for _, format := range []int16{
+			pgx.BinaryFormatCode,
+			pgx.TextFormatCode,
+		} {
+			tb.(*testing.T).Run(strconv.Itoa(int(format)), func(t *testing.T) {
+				var geom geom.T
+				err := conn.QueryRow(ctx, "select ST_SetSRID('POINT(1 2)'::geography, 4326)", pgx.QueryResultFormats{format}).Scan(&geom)
+				assert.NoError(t, err)
+				// assert.Equal(t, mustNewGeomFromWKT(t, "POINT(1 2)", 4326).SetSRID(4326).ToEWKBWithSRID(), geom.ToEWKBWithSRID())
```
Comment (twpayne) [inline pgxgeom_test.go]: This test can be updated to match 0043bb365a1e40322ec20c0ef6d1d551f9ec097e.

**Review by twpayne — COMMENTED** [review]
Thank you for another excellent PR!

Just one minor nit. Once you've made the changes, please could you squash all the commits in to one with the message:

    feat: add initial geography support

File: pgxgeom_test.go (hunk 1 above)
Comment (wchargin) [reply pgxgeom_test.go]: Done.

**Review by wchargin — COMMENTED** [review]
Thanks! Updated and squashed.

**Comment by twpayne** [comment]
Following #3, this PR needs a little conflict resolution. Would you be able to do it?

**Review by twpayne — APPROVED** [review]

**Comment by wchargin** [comment]
Certainly; done.

**Comment by wchargin** [comment]
Thanks!

**Comment by twpayne** [comment]
Thank you! I've tagged `v0.0.2` with this PR and #3 included.

**Pull request merged**